Remove makefile logic dealing with mysql.info.  We cannot ship that
documentation since it is not freely redistributable.


diff -Naur mysql-5.1.44.orig/Docs/Makefile.am mysql-5.1.44/Docs/Makefile.am
--- mysql-5.1.44.orig/Docs/Makefile.am	2010-02-04 06:36:50.000000000 -0500
+++ mysql-5.1.44/Docs/Makefile.am	2010-02-20 17:23:04.000000000 -0500
@@ -13,28 +13,21 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 
-EXTRA_DIST =		mysql.info INSTALL-BINARY @extra_docs@
+EXTRA_DIST =		INSTALL-BINARY @extra_docs@
 
-# make sure that "make install" installs the info page, too
-# automake only seems to take care of this automatically,
-# if we're building the info page from texi directly.
-install-data-hook:	$(srcdir)/mysql.info
+install-data-hook:
 	if test `basename $(prefix)` = "mysql" ; then \
 	  $(mkinstalldirs) $(DESTDIR)$(prefix)/docs ; \
-	  $(INSTALL_DATA) $(srcdir)/mysql.info $(DESTDIR)$(prefix)/docs ; \
 	  test ! -f $(top_srcdir)/ChangeLog || $(INSTALL_DATA) $(top_srcdir)/ChangeLog $(DESTDIR)$(prefix)/docs ; \
 	else \
-	  $(mkinstalldirs) $(DESTDIR)$(infodir) $(DESTDIR)$(pkgdatadir) ; \
-	  $(INSTALL_DATA) $(srcdir)/mysql.info $(DESTDIR)$(infodir) ; \
+	  $(mkinstalldirs) $(DESTDIR)$(pkgdatadir) ; \
 	  test ! -f $(top_srcdir)/ChangeLog || $(INSTALL_DATA) $(top_srcdir)/ChangeLog $(DESTDIR)$(pkgdatadir) ; \
 	fi
 
 uninstall-local:
 	if test `basename $(prefix)` = "mysql" ; then \
-	  @RM@ -f $(DESTDIR)$(prefix)/docs/mysql.info ; \
 	  @RM@ -f $(DESTDIR)$(prefix)/docs/ChangeLog ; \
 	else \
-	  @RM@ -f $(DESTDIR)$(infodir)/mysql.info ; \
 	  @RM@ -f $(DESTDIR)$(pkgdatadir)/ChangeLog ; \
 	fi
 
